### PR TITLE
Align progress on sequenial-challenge

### LINKED ISSUE #74: Align progress bar of sequential challenges with menu bar
When running a sequential challenge, we render an additional bar that shows how many tasks are there in total and how much time is remaining (not visible).

The _"Task n of m"_ part is not aligned with the bar above it. One of them is misaligned. Find out which, and make it look clean.

### PR DESCRIPTION
**Pull request by victorbalan**
fixes #74

### CODE REVIEW

File: app/elements/submission/sequential/sequential-challenge.html [hunk 1]
```diff
@@ -43,8 +43,8 @@
 			margin: 0;
 			padding-top: 10px;
 		}
-		#leftFill{
-			width: 50px;
+		#leftProgressFlex{
+			margin-right: 50px;
```
Comment (pbochis) [inline app/elements/submission/sequential/sequential-challenge.html:47]: will this work on different screen sizes?

Comment (lorenzleutgeb) [inline app/elements/submission/sequential/sequential-challenge.html:47]: This is specific to sequential challenges and not reused anywhere. If you want to code on your phone, please die.

Comment (victorbalan) [inline app/elements/submission/sequential/sequential-challenge.html:47]:
``` css
#taskTimer{
    margin: 0;
    padding-top: 10px;
    width: 50px;
 }
```

what i did is the counterpart of this. And why won`t this work on different screens? Do you know any screen that is smaller than 150px?

Comment (victorbalan) [inline app/elements/submission/sequential/sequential-challenge.html:47]: If you run coduno on that i`ll change the whole app.

**Pull request merged**